backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/process/ProcessInstanceResult.java

```diff
@@ -140,6 +142,7 @@ public boolean isSuccess()
 		private final ViewProfileId profileId;
 		@Builder.Default
 		private final boolean modalOverlay = true;
+		private final boolean openInNewTab;
```

Comment:
Instead of modalOverlay and openInNewTab flags, 
pls introduce an enum, called OpenTarget: SAME_TAB, SAME_TAB_OVERLAY, NEW_TAB.